packages/modules/create-itwin-viewer/bin.js

```diff
@@ -0,0 +1,184 @@
+  );
+  const editApp = bold(
+    cyan(
+      `${i++}. open the directory in your editor of choice to edit the application`
+    )
+  );
+  console.log(endMsg);
+  console.log(nextSteps);
+  console.log(chDir);
+  console.log(editApp);
+  console.log(startApp);
+}
+
+function installDependencies(appRoot, appName) {
+  const npmMsg = "Installing dependencies...";
+  console.log(npmMsg);
+  process.chdir(appRoot);
+  exec("npm install", () => {
+    finalize(appName);
+  });
+}
+
+function copyTemplate(appRoot, platform) {
+  const generatorRoot = new URL(".", import.meta.url).pathname;
+  const templateRoot = path.resolve(generatorRoot, "templates", platform);
+
+  fs.copySync(templateRoot, appRoot);
+}
+
+function writeConfig(appRoot, template, mergedAppConfig) {
+  const configFilePath = path.resolve(appRoot, "./src/config.ts");
+  let configFile = fs.readFileSync(configFilePath, "utf8");
+  const uiConfig = `export const uiConfig: UiConfiguration = ${JSON.stringify(
+    uiConfigurations[template].config
+  )}`;
+  const appConfig = `export const appConfig: AppConfiguration = ${JSON.stringify(
+    mergedAppConfig
+  )}`;
+  configFile = configFile.replace("// UI CONFIG HERE", uiConfig);
+  configFile = configFile.replace("// APP CONFIG HERE", appConfig);
+  fs.writeFileSync(configFilePath, configFile);
+}
+
+function writePackageJson(appRoot, appName) {
+  packageJson.name = appName;
+  packageJson.version = "0.1.0";
+  const packageJsonPath = path.resolve(appRoot, "./package.json");
+  fs.writeFileSync(packageJsonPath, JSON.stringify(packageJson));
+}
+
+async function main() {
+  const welcomeMsg = bold(
+    cyan("Welcome to the iTwin Viewer Application Generator!")
+  );
+  console.log(welcomeMsg);
+
+  const templates = [];
+
+  for (const template in uiConfigurations) {
+    templates.push({
+      title: uiConfigurations[template].description,
+      value: template,
+    });
+  }
+
+  const mainOptions = await prompts([
+    {
+      type: "select",
+      name: "platform",
+      message: "What type of application are you building?",
+      choices: [
+        { title: "Web", value: "web" },
+        { title: "Desktop", value: "desktop" },
+      ],
+    },
+    {
+      type: "text",
+      name: "name",
+      message: "What is the name of your application?",
+      validate: (val) =>
+        validatePackageName(val).validForNewPackages
+          ? true
+          : `${val} is not a valid package name. Please visit https://github.com/npm/validate-npm-package-name#naming-rules for guidelines`,
+    },
+    {
+      type: "select",
+      name: "template",
+      message: "Which template would you like to start with?",
+      choices: templates,
+    },
+    {
+      type: "text",
+      name: "iTwinId",
+      message: "Enter a default iTwinId",
+    },
+    {
+      type: "text",
+      name: "iModelId",
+      message:
+        "Enter a default iModelId (that is a part of the iTwin that you entered in the previous step)",
```

Comment:
Maybe change to "(_associated with the iTwinId_ that you entered in the previous step)" or similar?